src/OrchardCore.Modules/OrchardCore.Queries/Sql/SqlParser.cs

```diff
@@ -140,7 +140,7 @@ private string EvaluateSelectStatement(ParseTreeNode selectStatement)
 
             if (!String.IsNullOrEmpty(_where))
             {
-                sqlBuilder.WhereAlso(_where);
+                sqlBuilder.WhereAnd(_where);
```

Comment:
Could it be this line that makes it fail?

Reply:
Possible. Before making this change, I took a look at https://github.com/sebastienros/yessql/pull/272/files#diff-e3bd893e42f046e064d35d69baddd662R197, and it seems `WhereAnd` replaces `WhereAlso`, but I may be wrong.

Reply:
>  it seems WhereAnd replaces WhereAlso

That's what Seb tell but the question there are two versions now `WhereAnd` and `WhereOr` which the parser needs to differentiate between them, I will try to check this in my PR